tests/test_core.py

```diff
@@ -8131,6 +8131,10 @@ def test_asyncify_assertions(self):
     'unconditional': (False,),
   })
   def test_emscripten_lazy_load_code(self, conditional):
+    # don't minify JS whitespace, as it includes the entire program in the
+    # stack trace, which confuses our checking if the expected text is in
+    # the output
+    self.emcc_args += ['-g1']
```

Comment:
Would `['--minify', '0']` express this intent more precisely?

Reply:
Those are synonyms, yes. Would that be clearer for you? (I usually prefer the shorter notation myself...)